af/fastpath/fastpath/tests/test_unit.py

```diff
@@ -518,21 +518,35 @@ def test_score_torsf2():
     }
 
 
+def test_score_openvpn():
+    msm = loadj("torsf_2")
+    scores = fp.score_measurement(msm)
+    assert scores == {
+        "blocking_country": 0.0,
+        "blocking_general": 0.0,
+        "blocking_global": 0.0,
+        "blocking_isp": 0.0,
+        "blocking_local": 0.0,
+        "extra": {"bootstrap_time": 78.980935917, "test_runtime": 79.196301917},
+    }
+
+
+
+
 # # Bug tests
 
 
 def test_bug_backend351():
     # https://api.ooni.io/api/v1/measurement/temp-id-386770148
-    msm = loadj("bug_351")
+    msm = loadj("openvpn")
```

Comment:
Why was this changed to `openvpn` from `bug_351`, but the test is still called `test_bug_backend351`?